### ISSUE TITLE
feature support table?

### ISSUE THREAD

**Issue by johnwlockwood** [alex/alchimia#21]
Do you have a table of sqlalchemy.core features vs features supported by alchimia?

**Comment by brighid** [comment]
+1: The README saying only that you can "use most of the SQLAlchemy-core API [but not] the ORM" is anything but confidence-inspiring. A specific list of what's supported would make the library tremendously easier to use—"installing our thing and writing code against it is the *only way* to find out if it suits your use case at all" is one of the classic OSS UX antipatterns. :(

**Comment by glyph** [comment]
I agree. Unfortunately, I'm not quite sure how to tell which bits of documentation from SQLAlchemy itself will tell me where to look.

Would it be better to adjust the claim to just be "SQLAlchemy-core but not the ORM", since as far as I know that's what we believe it to be, and then add any other things we discover don't work as we discover them?

**Comment by johnwlockwood** [comment]
Yes. @glyph, being confident about your belief it supports SQLAlchemy-core would be a service.
